mscore/albummanager.h

```diff
@@ -13,39 +13,118 @@
 #ifndef __ALBUMMANAGER_H__
 #define __ALBUMMANAGER_H__
 
+#include <optional>
```

Comment:
macos < 10.14 doesn't have full c++17 support, need to back out optional use.
See https://github.com/octobanana/peaclock/issues/2